app/src/main/java/gov/hhs/cdc/trustedintermediary/auth/AuthDomainRegistration.java

```diff
@@ -20,12 +20,14 @@
  */
 public class AuthDomainRegistration implements DomainConnector {
 
+    public static final String AUTH_API_ENDPOINT = "/v1/auth";
```

Comment:
You can make these "package private" which will allow them to be used in tests but not outside the tests.

```suggestion
    static final String AUTH_API_ENDPOINT = "/v1/auth";
```